nemo_aligner/models/nlp/gpt/megatron_gpt_critic.py

```diff
@@ -122,7 +122,7 @@ def fwd_output_and_loss_func(data_iterator, model):
             mask = batch["mask"]
 
             attention_mask, _, position_ids = get_ltor_masks_and_position_ids(
-                tokens, self.tokenizer.eos_id, False, True, False,
+                tokens.cuda(), self.tokenizer.eos_id, False, True, False,
```

Comment:
I think it's better to just let the tokens be on cuda(i.e

```
tokens = batch["tokens"].cuda()
```

Reply:
tested, done